capabilities/types/types.ipldsch

```diff
@@ -1,4 +1,24 @@
 type Headers {String:String}
 type Digest struct {
   digest Bytes
-}
\ No newline at end of file
+}
+
+type Await struct {
+  selector String
+  link Link
+} representation tuple
+
+type Promise struct {
+  ucanAwait Await (rename "ucan/await")
+}
+
+type Multihash bytes
+type Multiaddr bytes
+type HasMultihash any
+type DID bytes
+type URL string
+type HTTPHeaders any
```

Comment:
We have a `Headers` type defined as a map above, would it be possible to unify around a common type in the future? I think golangs headers are usually `map[string][]string`.